### PR TITLE
LRSD-6871 Fixing responsiveness issue of the mobile layout

### PR DESCRIPTION
**Pull request by GabrielFernandoS**

### CODE REVIEW

**Comment by nNilton** [comment]
ci:test:sf

**Comment by rtienpal** [comment]
Please sort (576px goes in front of768px)

**Comment by lluk4z** [comment]
ci:test:sf

File: workspaces/liferay-learn-workspace/client-extensions/liferay-learn-global-css/assets/global.css [hunk 1]
```diff
@@ -1483,12 +1483,22 @@ pre[class*='language-'] {
...
-	}
+.page-tabs {
+  @media (max-width: 576px) {
+    .navbar-nav .nav-item {
+      width: 50%;
+    }
+    .navbar-nav .nav-item .navbar-text-truncate span {
+      white-space: normal;
+      display: flex;
+      line-height: normal;
```
Comment (nNilton) [inline workspaces/liferay-learn-workspace/client-extensions/liferay-learn-global-css/assets/global.css:1494]: please sort these items alfabetically

**Review by nNilton — CHANGES_REQUESTED** [review]

**Comment by nNilton** [comment]
ci:test:sf

**Comment by nNilton** [comment]
ci:test:sf

**Comment by nNilton** [comment]
resent:https://github.com/brianchandotcom/liferay-portal/pull/155538

**Pull request closed without merging**